### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZigRazor**
Update Readme to notify of the API changes, and announcing the next official major release on 1st November.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ Share on [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=
 **CXXGraph** is a small library, header only, that manages the Graph and it's algorithms in **C++**. In other words a "Comprehensive C++ Graph Library".
 An alternative to [Boost Graph Library (BGL)](https://www.boost.org/doc/libs/1_77_0/libs/graph/doc/index.html).
 
+## Warning ⚠️
+
+*From the official release v2.0.0 we make some API break changes.*
```
Comment (sbaldu) [inline README.md:31]: Maybe "API breaking changes" would be more correct?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ Share on [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=
...
+
+A new major will be released after the Hacktoberfest, on 1st November. 📅
+
+In the meantime the main branch is updated with the last API interface.
```
Comment (sbaldu) [inline README.md:35]: Latest

Comment (ZigRazor) [reply README.md]: I correct it!

File: README.md (hunk 1 above)
Comment (ZigRazor) [reply README.md]: I correct it!

**Review by sbaldu — APPROVED** [review]

**Comment by sbaldu** [comment]
Perfect

**Pull request merged**